dev-util/bear/bear-3.1.3.ebuild

```diff
@@ -55,6 +55,7 @@ src_prepare() {
 }
 
 src_configure() {
+	export CMAKE_BUILD_PARALLEL_LEVEL=$(makeopts_jobs)
```

Comment:
Might be worth filing a bug for the eclass to do this too, with an explanation of why it was needed here, even if it is kind of gross to have to.